src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,6 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+  if (boolean === 10) return "Yes";
+  return "No";
```

Comment:
This is pretty flawed and should be rewritten. The purpose of the test is to pass in a boolean value (so either `true` or `false`) and then return a string of either Yes or No respectively. Here you are only testing if the number 10 is passed in and nothing more
```
  return boolean === true ? "Yes" : "No";
```
This is the correct function, but your test as it stands will fail as it was testing the flawed logic here